magnet_schema/src/support.rs

```diff
@@ -56,6 +56,14 @@ pub fn extend_schema_with_bounds(mut schema: Document, bounds: Bounds) -> Docume
     schema
 }
 
+#[doc(hidden)]
```

Comment:
(Therefore, this function will not be necessary, its body will be incorporated into `extend_schema()`.)